ucm2/sof-soundwire/rt715.conf

```diff
@@ -1,20 +1,54 @@
 # Use case Configuration for sof-soundwire card
+#
+Syntax 3
+Define.KernelModulePath "class/leds/privacy\:\:micmute"
+Define.MicMute "$${sys:$KernelModulePath}"
 
-SectionDevice."Mic" {
-	Comment	"SoundWire microphones"
+If.driver {
+	Condition {
+		Type String
+		Empty "${var:MicMute}"
```

Comment:
@GoPerry The rt715 patch is merged https://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git/commit/?h=for-5.13&id=dcca646c4977d885af3466b454df97b9cb0e0d26 

However, from the patch, 'rt715 Main Capture' is not related to the "class/leds/privacy\:\:micmute". This means even without "class/leds/privacy\:\:micmute", 'rt715 Main Capture' still can be used. Or even with "class/leds/privacy\:\:micmute", 'rt715 Main Capture' may not be supported. It is better to test 'rt715 Main Capture' itself, such as "ControlExists". Please refer: https://github.com/alsa-project/alsa-ucm-conf/blob/master/ucm2/sof-hda-dsp/sof-hda-dsp.conf